pkg/position_store/position.go

```diff
@@ -28,47 +28,44 @@ type Position struct {
 	// Version is the schema version of position
 	Version string
 	// Name is the unique name of a pipeline
-	Name       string
-	Stage      config.InputMode
-	Value      interface{}
-	UpdateTime time.Time
+	Name        string
+	Stage       config.InputMode
+	Value       interface{} `bson:"-" json:"-"`
+	ValueString string      `bson:"value" json:"value"`
```

Comment:
Once have a position, it's hard to remember whether it has a value or a value string or both. I think the previous design is better.

Reply:
@Ryan-Git updatd another design:

Extract `PositionMeta` and changed PositionRepo's interface.